ipt_connect/scripts/jurystats.py

```diff
@@ -8,10 +8,10 @@
 os.environ['DJANGO_SETTINGS_MODULE'] = "ipt_connect.settings"
 django.setup()
 
-from IPT2016.models import *
+from IPTdev.models import *
 jurys = Jury.objects.all()
 
-
+#TODO: adapt this for any tournament, not only IPT216
```

Comment:
todo unclear, since you removed the IPT2016 dependency ?

Reply:
The removal could be not enough, I haven't checked this (because I don't know what the script does).

How about rewriting TODO this way:
`make sure if it works for any tournament, not only IPT2016`.

Another option is just to remove the commit.

Reply:
Ok, I see. Rewriting the todo as you propose should be fine. Better safe than sorry.